website/content/docs/connect/cluster-peering/index.mdx

```diff
@@ -1,32 +1,58 @@
 ---
 layout: docs
-page_title: Service Mesh - What is Cluster Peering?
+page_title: Cluster Peering Overview
 description: >-
-  Cluster peering establishes communication between independent clusters in Consul, allowing services to interact across datacenters. Learn about the cluster peering process, differences with WAN federation for multi-datacenter deployments, and technical constraints.  
+  Cluster peering establishes communication between independent clusters in Consul, allowing services to interact across datacenters. Learn how cluster peering works, its differences with WAN federation for multi-datacenter deployments, and how to troubleshoot common issues.  
 ---
 
-# What is Cluster Peering?
+# Cluster peering overview
 
-You can create peering connections between two or more independent clusters so that services deployed to different partitions or datacenters can communicate.
+This topic provides an overview of cluster peering, which is a process that connects two or more independent clusters so that services deployed to different partitions or datacenters can communicate.
 
-## Overview
+Cluster peering is enabled in Consul by default. For specific information about cluster peering configuration and usage, refer to following pages.
 
-Cluster peering is a process that allows Consul clusters to communicate with each other. The cluster peering process consists of the following steps:
+**Reference information:**
 
-1. Create a peering token in one cluster.
-1. Use the peering token to establish peering with a second cluster.
-1. Export services between clusters.
-1. Create intentions to authorize services for peers.
+- [HTTP API reference: `/peering/` endpoint](/consul/api-docs/peering)
+- [CLI reference: `peering` command](/consul/commands/peering).
+- [Cluster peering technical specifications](/docs/consul/cluster-peering/configuration)
 
-This process establishes cluster peering between two [admin partitions](/consul/docs/enterprise/admin-partitions). Deployments without an Enterprise license can still use cluster peering because every datacenter automatically includes a `default` partition.
+**Usage information:**
 
-For detailed instructions on establishing cluster peering connections, refer to [Create and Manage Peering Connections](/consul/docs/connect/cluster-peering/create-manage-peering).
+- [Establish cluster peering connections](/docs/connect/cluster-peering/create-manage-peering)
+- [Manage cluster peering connections](/consul/docs/cluster-peering/usage/manage-connections)
+- [L7 traffic management for cluster peering connections](/consul/docs/cluster-peering/usage/peering-traffic-management)
```

Comment:
some of these links are wrong, can you verify that all cluster peering links start wtih `/consul/docs/connect`?

```suggestion
- [Establish cluster peering connections](/consul/docs/connect/cluster-peering/create-manage-peering)
- [Manage cluster peering connections](/consul/docs/connect/cluster-peering/usage/manage-connections)
- [L7 traffic management for cluster peering connections](/consul/docs/connect/cluster-peering/usage/peering-traffic-management)
```